### PR TITLE
fix(curriculum): fixed typo for clarity

### PR DESCRIPTION
**Pull request by andrewmchoward**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

updated language from "cat nip" --> "catnip"

### CODE REVIEW

**Review by jdwilkin4 — COMMENTED** [review]
Hi @andrewmchoward ! 

Thank you for your interest in contributing to freeCodeCamp. 

This change really should be applied to all of the challenges in the basic css legacy block that use this cat example.

At quick glance that looks like that involves making changes to the first challenge all the way through to the last one here 
https://www.freecodecamp.org/learn/responsive-web-design/basic-css/use-an-id-attribute-to-style-an-element

**Comment by andrewmchoward** [comment]
Thank you, I have updated the remaining ones that I could locate!

File: catnip
Comment (gikf) [inline catnip:1]: This file needs to be removed from the PR

**Review by majestic-owl448 — APPROVED** [review]
LGTM 👍

**Review by Sembauke — APPROVED** [review]

**Pull request merged**